content/rest-api/artifacts.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Artifacts API
+description: Authenticated and public access to build artifacts
+weight: 3
+---
+
+## Authenticated download URL
+
+`GET /artifacts/:uuid1/:uuid2/:artifactName`
+
+This URL can be obtained using the [Builds API](/rest-api/builds) or copied directly from the Codemagic UI.
+
+## Retrieve public download URL
+
+`POST /artifacts/:uuid1/:uuid2/:artifactName/public-url`
+
+#### Parameters
```

Comment:
This should be ###

Reply:
It's like that everywhere on the [/rest-api/builds.md](https://github.com/codemagic-ci-cd/codemagic-docs/blob/master/content/rest-api/builds.md?plain=1#L17) page